tests/integration/test_status.bats

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bats
+# Integration tests for status command
```

Comment:
Why do we have two test files for the same regression? Shouldn't there be just one?